libraries/c_sdk/standard/mqtt/include/iot_mqtt.h

```diff
@@ -256,13 +256,13 @@ void IotMqtt_ReceiveCallback( void * pNetworkConnection,
  *
  * // Example network abstraction types.
  * IotNetworkServerInfo_t serverInfo = { ... };
- * IotNetworkCredentialInfo_t credentialInfo = { ... };
+ * cIotNetworkCredentials_t credentialInfo = { ... };
```

Comment:
I don't think this change is needed.  Could you revert this line?  Thanks.

Reply:
I've corrected misspelling (removing first c letter). 
Also cannot revert cause there is no such struct in AWS FreeRTOS. 
[Here](https://github.com/aws/amazon-freertos/blob/master/demos/defender/aws_iot_demo_defender.c#L207) you can check sample use of this structure